real_estate/models/estate_property.py

```diff
@@ -0,0 +1,232 @@
+    )
+    facades = fields.Integer(
+        string="Number of Facades",
+        required=True,
+        help="Enter the number of facades the property has"
+    )
+    garage = fields.Boolean(
+        string="Has Garage",
+        help="Check if the property has a garage"
+    )
+    garden = fields.Boolean(
+        string="Has Garden",
+        help="Check if the property has a garden"
+    )
+    garden_area = fields.Integer(
+        string="Garden Area (sq meters)",
+        help="Enter the area of the garden"
+    )
+    garden_orientation = fields.Selection(
+        selection=[
+            ('north', "North"),
+            ('south', "South"),
+            ('east', "East"),
+            ('west', "West")
+        ],
+        string="Garden Orientation",
+        help="Select the orientation of the garden"
+    )
+    active = fields.Boolean(
+        string="Active",
+        default=True,
+        help="If unchecked, it will archive the property."
+    )
+    state = fields.Selection(
+        selection=[
+            ('new', "New"),
+            ('offer_received', "Offer Received"),
+            ('offer_accepted', "Offer Accepted"),
+            ('sold', "Sold"),
+            ('cancelled', "Cancelled"),
+        ],
+        string="State",
+        required=True,
+        default='new',
+        copy=False,
+        help="Current state of the property."
+    )
+    date_of_deadline = fields.Date(
+        string="Date Deadline",
+        copy=False,
+        default=lambda self: fields.Date.today()+timedelta(days=10),
+        help="Date when the property will not be available",
+        compute="_compute_property_deadline"
+    )
+
+    property_type_id = fields.Many2one(
+        comodel_name='estate.property.type',
+        string="Property Type"
+    )
+    buyer_id = fields.Many2one(
+        comodel_name='res.partner',
+        string="Buyer",
+        copy=False,
+        domain="[('is_company', '=', True)]"
+    )
+    salesperson_id = fields.Many2one(
+        comodel_name='res.users',
+        string="Salesperson",
+        default=lambda self: self.env.user
+    )
+    company_id = fields.Many2one(
+        comodel_name='res.company',
+        string="Company",
+        default=lambda self: self.env.company
+    )
+
+    tag_ids = fields.Many2many(
+        comodel_name='estate.property.tag',
+        string="Tag"
+    )
+
+    offer_ids = fields.One2many(
+        comodel_name='estate.property.offer',
+        inverse_name='property_id',
+        string="Offers"
+    )
+
+    total_area = fields.Float(
+        string="Total Area",
+        compute='_compute_total_area'
+    )
+    best_price = fields.Float(
+        string="Best Offer",
+        compute='_compute_best_price',
+    )
+
+    @api.depends('garden_area', 'living_area')
+    def _compute_total_area(self):
+        for property in self:
+            property.total_area = property.garden_area + property.living_area
+    @api.depends('offer_ids.price')
```

Comment:
```suggestion
            property.total_area = property.garden_area + property.living_area

    @api.depends('offer_ids.price')
```
Please add a new empty line between two methods 🙏🏻